.gitmodules

```
@@ -0,0 +1,4 @@
+[submodule "libvyosconfig"]
+	path = libvyosconfig
+	url = git@github.com:vyos/libvyosconfig.git
```

Please switch to a relative path here

```suggestion
	url = ../libvyosconfig.git
```